aloha-core/src/main/scala/com/eharmony/aloha/models/exploration/BootstrapModel.scala

```diff
@@ -78,9 +79,10 @@ case class BootstrapModel[A, B](
     successes: sci.IndexedSeq[Int] = sci.IndexedSeq.empty)(implicit audit: Boolean): (ModelOutput[B], Option[Score]) = {
     if (models.isEmpty) {
       val decision = explorer.chooseAction(salt(a), successes)
+      val usedSubScores = subScores.filter(_.getScore.getExtension(IntScore.impl).getScore == decision.getAction)
```

Comment:
I don't like this line at all, but I'm not sure of the best way to get either an Int or Long subscore, @deaktator, what do you think?